pyfar/classes/audio.py

```diff
@@ -1021,7 +1039,122 @@ def power(data: tuple, domain='freq'):
     return _arithmetic(data, domain, _power)
 
 
-def _arithmetic(data: tuple, domain: str, operation: Callable):
+def matrix_multiplication(
+        data: tuple, domain='freq', axes=[(-2, -1), (-2, -1), (-2, -1)]):
+    """Matrix multiplication of multidimensional pyfar audio objects and/or
+    array likes.
+
+    This function is based on ``numpy.matmul`` and only acts on the channels,
+    not on the frequency/time dimension. Pyfar audio objects are:
```

Comment:
I'm not sure if this makes it clear. Maybe make more clear that the number of samlpes/frequency bins always stays the same.